icons/ton.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Ton</title><path d="M8.4 9c-.1 0-.2.1-.2.2s-.1.1 0 .2c0 0 0 .1.1.3l3.2 5.6V9H8.4zm7.3.1c0-.1 0-.1 0 0-.1-.1-.2-.1-.3-.1h-2.9v6.3l3.3-5.6c0-.1.1-.2.1-.3 0-.1 0-.2-.1-.2 0-.1-.1-.1-.1-.1zM12 0C5.4 0 0 5.4 0 12s5.4 12 12 12 12-5.4 12-12S18.6 0 12 0zm4.7 10.2-4.3 7.3c-.1.2-.3.3-.5.3s-.4-.1-.5-.3l-4.2-7.3c.1-.2-.1-.4-.1-.7 0-.3 0-.6.2-.8.1-.3.4-.5.6-.6.3-.1.6-.1.7-.1h6.7c.2 0 .5 0 .7.2.3.1.5.4.6.5.1.2.2.5.2.8.1.1 0 .4-.1.7z"/></svg>
```

Comment:
I'm seeing major differences between your version and the SVG symbol found in their brand assets. Can you take another look at it?
![image](https://github.com/simple-icons/simple-icons/assets/12817388/2a652ff8-3027-44b9-87b5-7d4f1d04bf57)